amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/QueueContainer.java

```diff
@@ -15,21 +15,43 @@
 package io.streamnative.pulsar.handlers.amqp;
 
 import com.google.common.collect.Maps;
+import io.streamnative.pulsar.handlers.amqp.impl.PersistentQueue;
 import java.util.Map;
+import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
 import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.lang3.StringUtils;
+import org.apache.pulsar.broker.PulsarServerException;
+import org.apache.pulsar.broker.PulsarService;
+import org.apache.pulsar.broker.service.Topic;
+import org.apache.pulsar.broker.service.persistent.PersistentTopic;
 import org.apache.pulsar.common.naming.NamespaceName;
 
 /**
  * Container for all queues in the broker.
  */
+@Slf4j
 public class QueueContainer {
+    private Executor executor = Executors.newCachedThreadPool();
```

Comment:
Seem as the above comment. we don't need to introduce a new thread pool here.

Reply:
Done.